rcl_action/include/rcl_action/action_client.h

```diff
@@ -741,6 +742,56 @@ bool
 rcl_action_client_is_valid(
   const rcl_action_client_t * action_client);
 
+/// Return the feedback subscription for a rcl_action_client_t.
+/**
+ * This function returns the action client's internal feedback subscription.
+ * This function can fail, and therefore return `NULL`, if the:
+ *   - action client is `NULL`
+ *   - action client is invalid (never called init, called fini, or invalid)
+ *
+ * The returned struct is only valid as long as the action client is valid.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
```

Comment:
If only get pointer, is it thread-safe ?

Reply:
It's not thread-safe (almost all of the api in this files are not thread-safe), so the outside called need to add a lock for it.

Reply:
My mistake. It's not thread-safe.